controllers/C_InsuranceCompany.class.php

```diff
@@ -30,9 +30,9 @@ public function default_action()
 
     public function edit_action($id = "", $patient_id = "", $p_obj = null)
     {
-        if ($p_obj != null && get_class($p_obj) == "insurancecompany") {
+        if ($p_obj != null && $p_obj::class == "insurancecompany") {
```

Comment:
I can't figure out how this was or is working right. Don't both of those mechanisms return InsuranceCompany for the class name? If so, then how can they equal the lowercase insurancecompany?

Reply:
it looks like the controller class doesn't send `$p_obj` so is always null.

Reply:
ref #8998